src/client.js

```diff
@@ -64,6 +64,7 @@ function Client(manager, name, config = {}) {
 		highlightRegex: null,
 		highlightExceptionRegex: null,
 		messageProvider: undefined,
+		expectedPings: {},
```

Comment:
this seems to be the wrong level... we need to track such things per the network they originate from and not globally per client